food-truck-frontend/src/components/theme/ThemeD2P.ts

```diff
@@ -0,0 +1,43 @@
+import {Dispatch} from "redux";
+import {ThemeAction, ThemeActionTypes} from "../../redux/theme/ThemeActions";
+
+function getStoredTheme(): boolean {
```

Comment:
I'll confess, the name doesn't make much sense to me. The function looks like it's checking for dark mode, not getting the stored theme.

Reply:
I'll change that really quick. It was meant to return a string initially but I changed it to a boolean.